src/coreclr/pgosupport.cmake

```diff
@@ -24,6 +24,15 @@ function(add_pgo TargetName)
             if(UPPERCASE_CMAKE_BUILD_TYPE STREQUAL RELEASE OR UPPERCASE_CMAKE_BUILD_TYPE STREQUAL RELWITHDEBINFO)
                 target_compile_options(${TargetName} PRIVATE -flto -fprofile-instr-generate)
                 set_property(TARGET ${TargetName} APPEND_STRING PROPERTY LINK_FLAGS " -flto -fprofile-instr-generate")
+                if(CMAKE_CROSSCOMPILING AND CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+                    string(REPLACE "." ";" CLANG_VERSION "${CMAKE_C_COMPILER_VERSION}")
```

Comment:
nit: we can use same language in these lines for consistency: `_C_` or `_CXX_`

Reply:
Oops, I've missed that, thanks for spotting it.